x/metadata/client/cli/cli_test.go

```diff
@@ -1471,9 +1471,13 @@ func (s *IntegrationCLITestSuite) TestGetMetadataRecordSpecCmd() {
 			expOut: []string{indent(s.recordSpecAsText, 4)},
 		},
 		{
-			name:   "rec spec id does not exist",
-			args:   []string{"recspec1qh00d0q2e8w5say53afqdesxp2zw42dq2jdvmdazuwzcaddhh8gmuqhez44"},
-			expOut: []string{"record_specifications: []"},
+			name: "rec spec id does not exist",
+			args: []string{"recspec1qh00d0q2e8w5say53afqdesxp2zw42dq2jdvmdazuwzcaddhh8gmuqhez44"},
+			expOut: []string{
+				"      contract_spec_addr: contractspec1q000d0q2e8w5say53afqdesxp2zqzkr4fn",
+				"    record_spec_addr: recspec1qh00d0q2e8w5say53afqdesxp2zw42dq2jdvmdazuwzcaddhh8gmuqhez44",
+				"  specification: null",
+			},
```

Comment:
> The indentation levels and field names do not align with the actual response format...

Yes they do. You can tell they align with the actual response format because the test passes.

The output that it's being tested against (in this test case) is YAML with proper formatting. The extra spaces at the start of each `expOut` entry reflects the level that each has in the YAML output. Note that not all of the output is listed in `expOut`, only a few lines that demonstrate that it's making the right request and getting the expected "null" specification result.